velox/exec/Exchange.cpp

```diff
@@ -135,6 +135,10 @@ void Exchange::close() {
   exchangeClient_ = nullptr;
 }
 
+uint64_t Exchange::backgroundCpuTimeMs() const {
+  return exchangeClient_ ? exchangeClient_->backgroundCpuTimeMs() : 0;
```

Comment:
Shall we check exchangeClient_ is not null and we don't expect call this after operator close? thanks!

Reply:
Thank you for the suggestions @xiaoxmeng! Exchange::close can potentially be called twice, that's why exchangeClient_ can be null. I think returning 0 in this case would be correct as the previously collect backgroundTime is already folded into finishTime by then. Do you feel we should explicitly track isClosed for this operator?